src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+package duke;
+
+import duke.exception.EmptyDescription;
+import duke.keycommand.Deadline;
+import duke.keycommand.Event;
+import duke.keycommand.ToDo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    private static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+    private static final String INSTRUCTION = "Please give your command in the following format";
+    private static final String HorizontalLine = "____________________________________________________________";
```

Comment:
Good use of named constants in your code! You might want to move some of them into another file to better structure your code. Also, there is a small naming issue on this particular named constant.